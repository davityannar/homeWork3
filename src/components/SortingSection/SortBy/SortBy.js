import React from 'react'

import './SortBy.scss'
import Types from '../../Search/SearchTypes/Types/Types';

const SortBy = () => {
  return (
    <div className='sort-filters'>
      <p>
        Sort by
      </p>
      <Types inputId='release' inputClass='secondary-filter' inputName='sort' inputType='radio' inputValue='RELEASE' inputText='release date' checkedByDefault />
      <Types inputId='rating' inputClass='secondary-filter' inputName='sort' inputType='radio' inputValue='RATING' inputText='rating' />
    </div>
  )
}

export default SortBy;