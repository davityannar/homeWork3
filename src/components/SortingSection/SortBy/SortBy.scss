
.sort-filters {
  display: flex;

  p {
    margin: 0;
  }

  * {
    font-weight: bold;
  }
}