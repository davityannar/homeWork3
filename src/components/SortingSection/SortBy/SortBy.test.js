import React from 'react'
import SortBy from './SortBy'

describe('Tests SortBy component', () => {
  it('Matching snapshot', () => {
    const component = shallow(<SortBy />)

    expect(component).toMatchSnapshot();
  })
})