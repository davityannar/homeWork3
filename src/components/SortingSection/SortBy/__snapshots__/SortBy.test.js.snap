// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests SortBy component Matching snapshot 1`] = `
<div
  className="sort-filters"
>
  <p>
    Sort by
  </p>
  <Types
    checkedByDefault={true}
    inputClass="secondary-filter"
    inputId="release"
    inputName="sort"
    inputText="release date"
    inputType="radio"
    inputValue="RELEASE"
  />
  <Types
    inputClass="secondary-filter"
    inputId="rating"
    inputName="sort"
    inputText="rating"
    inputType="radio"
    inputValue="RATING"
  />
</div>
`;
